package.json

```diff
@@ -27,7 +27,12 @@
     "stylelint-config-standard": "^36.0.1",
     "stylelint-order": "^6.0.4",
     "turbo": "^2.4.4",
-    "typescript": "^5.7.3",
+    "typescript": "~5.7.3",
     "vitest": "^3.0.8"
+  },
+  "pnpm": {
+    "patchedDependencies": {
+      "vite-plugin-sass-dts": "patches/vite-plugin-sass-dts.patch"
+    }
```

Comment:
the plugin emitted extensions that didnt type check in with esm projects, patched to switch from `css.d.ts` to typescript supported `d.css.ts`.